src/OpenLoco/src/Windows/ObjectSelectionWindow.cpp

```diff
@@ -52,7 +52,53 @@ namespace OpenLoco::Ui::Windows::ObjectSelectionWindow
     static constexpr int kRowHeight = 12;
     static constexpr Ui::Size kWindowSize = { 600, 398 };
 
-    static loco_global<uint8_t[999], 0x004FE384> _4FE384;
+    enum ObjectTabFlags
+    {
+        none = 0,
+        alwaysHidden = 1 << 0,
+        advanced = 1 << 1,
+        hideInGame = 1 << 2,
+        hideInEditor = 1 << 3,
+        showEvenIfSingular = 1 << 4,
```

Comment:
This appears to be used only by the track and road mod tabs. The intent was apparently for them to never be hidden, even if only one mod is available.